doc/cijobs.md

```diff
@@ -0,0 +1,60 @@
+# CI jobs information
```

Comment:
It would be good to show somewhere how to use the cijobs command to avoid needing to build your own baseline.  Maybe that belongs in the "asm diffs" doc and this one should link to it.